br/pkg/task/backup_ebs.go

```diff
@@ -87,9 +123,15 @@ func RunBackupEBS(c context.Context, g glue.Glue, cfg *BackupConfig) error {
 	ctx, cancel := context.WithCancel(c)
 	defer cancel()
 
+	cleanUp, err := checkConflictingTasksAndRegisterSelf(ctx, cfg.Config)
```

Comment:
Blocking here is improper. We'd better to keep move it to the task of suspend gc and schedule phase. Then we can release as soon as possible.